source/MaterialXTest/MaterialXRuntime/Runtime.cpp

```diff
@@ -47,6 +47,39 @@ namespace mx = MaterialX;
 
 namespace
 {
+    struct RuntimeGlobals
+    {
+        static const mx::FilePath& TARGETS_PATH()
+        {
+            static const mx::FilePath TARGET_PATH("targets");
+            return TARGET_PATH;
+        }
```

Comment:
If static usage of FilePath is the problem it would be better to fix the core of the issue inside the FilePath class instead. The use of global variables inside that class can probably be moved into the methods using it instead.